app/modules/usage_stats/migrations/2018_02_08_000001_usage_stats.php

```diff
@@ -0,0 +1,260 @@
+<?php
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Capsule\Manager as Capsule;
+
+class UsageStats201828000001 extends Migration
+{
+    
+    private $tableName = 'usage_stats';
+    private $tableNamebackup = 'usage_stats_backup';
+    
+    public function up()
+    {
+        $capsule = new Capsule();
+        $migrateData = false;
+        
+        if ($capsule::schema()->hasTable($this->tableName)) {
```

Comment:
This will always be true if the previous migration has run?

Reply:
migratedata should check if a column has the correct type?

Reply:
It'd be false if the module is initializing the table.